src/js/background/backgroundLogic.js

```diff
@@ -362,6 +364,17 @@ const backgroundLogic = {
   cookieStoreId(userContextId) {
     if(userContextId === 0) return "firefox-default";
     return `firefox-container-${userContextId}`;
+  },
+
+  async setDefaultOptions() {    
```

Comment:
I think `setDefaultOptions` is a bit too generic for a name and we don't really have other defaults right now. IMO, something like `setSyncExclusion` or the like would be more appropriate.